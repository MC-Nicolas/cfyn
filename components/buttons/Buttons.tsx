import React from 'react';
import styles from '../../styles/Buttons.module.css';

type Props = {
  title: string;
  backgroundCol: string;
  textColor: string;
};

export const ButtonLogin = ({ title, backgroundCol, textColor }: Props) => {
  return (
    <button
      className={styles.button1}
      style={{ backgroundColor: backgroundCol, color: textColor }}
    >
      {title}
    </button>
  );
};
