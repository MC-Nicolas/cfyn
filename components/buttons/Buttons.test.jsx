import React from 'react';
import { ButtonLogin } from './Buttons';

test('<ButtonLogin />', () => {
  const { debug, getByTestId } = render(
    <ButtonLogin
      title='Log in'
      backgroundCol='#000'
      textColor='#FFF'
      onClick={() => console.log('Has been clicked')}
    />
  );
  debug();
});
