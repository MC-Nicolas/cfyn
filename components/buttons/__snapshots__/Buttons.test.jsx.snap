// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button1 /> 1`] = `
<button
  data-testid="button01"
  style="background-color: rgb(255, 255, 255); color: rgb(0, 0, 0);"
>
  Log in
</button>
`;
